scripts/set-version.js

```diff
@@ -0,0 +1,163 @@
+const path = require('path');
+const replace = require('replace-in-file');
+const moment = require('moment');
+const {
+  displayConfirmationMessage,
+  displayErrorMessage
+} = require('./common');
+const hotPackageJson = require('../package.json');
+
+const CURRENT_VERSION = hotPackageJson.version;
+const SEMVER_SECTIONS = ['major', 'minor', 'patch'];
+
+/**
+ * Check if the provided version number is a valid semver version number.
+ *
+ * @param {string} version Version number.
+ * @returns {boolean} `true` if the version number is a valid semver version number, `false` otherwise.
+ */
+function isVersionValid(version) {
+  const semverRegex = new RegExp(
+    '^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)' +
+    '(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$'
+  );
+
+  return semverRegex.test(version);
+}
+
+/**
+ * Check if the provided release date in a format of 'DD/MM/YYYY' is a valid future date.
+ *
+ * @param {string} date The date in format of 'DD/MM/YYYY'.
+ * @returns {boolean} `true` if the release date is valid, `false` otherwise.
+ */
+function isReleaseDateValid(date) {
+  const dateSplit = date.split('/');
+  const now = moment();
+  const dateObj = moment({
+    day: dateSplit[0],
+    month: dateSplit[1] - 1,
+    year: dateSplit[2],
+  });
+
+  return dateObj.isAfter(now);
+}
+
+/**
+ * Set the provided version number to the packages' package.json files.
+ *
+ * @param {string} version The version number.
+ */
+function setVersion(version) {
```

Comment:
I think that there can be a problem when we want to release HoT as a beta. For example release, `10.0.0-beta1`. In this case, the Handsontable and its wrappers are bumped to the new version but additionally, the peer dependencies are bumped to the `^10.0.0` (or `>=10.0.0`) version. This would cause an error after publishing the wrappers.